Acumatica REST API Console Application/ODataExample.cs

```diff
@@ -2,79 +2,94 @@
 
 using Acumatica.Auth.Api;
 using Acumatica.RESTClient.Api;
+using Acumatica.RESTClient;
 
 using static Acumatica.Auth.Api.AuthApi;
 
 using Version = Acumatica.RESTClient.ODataVersion;
+using Acumatica.RESTClient.Client;
 
 namespace AcumaticaRestApiExample
 {
     public class ODataExample
 		
 	{
 		
+		
 		public static void OauthExample(string siteURL, string username, string password, string clientSecret, string clientID, string tenant = null)
 		{
 			Console.WriteLine("OData with Oauth authentication");
             var authApi = new AuthApi(siteURL,
 				requestInterceptor: RequestLogger.LogRequest, responseInterceptor: RequestLogger.LogResponse);
 			var configuration = authApi.ReceiveAccessToken(clientID, clientSecret, username, password, OAuthScope.API);
 			
-			ODataAPI example1 = new ODataAPI(configuration, Version.OData);
-			var response = example1.Get();
+			ODataAPI oauth = new ODataAPI(configuration, Version.OData);
```

Comment:
I don't like the "oauth" name. It's not descriptive enough. 
Please rename to something like "odataAPIWithOAuth"